internal/devconfig/configfile/env.go

```diff
@@ -25,10 +26,15 @@ func (c *ConfigFile) ParseEnvsFromDotEnv() (map[string]string, error) {
 	if !c.IsdotEnvEnabled() {
 		return nil, fmt.Errorf("env file does not have a .env extension")
 	}
-
-	file, err := os.Open(c.EnvFrom)
+	envFileAbsPath := filepath.Dir(c.AbsRootPath)
+	if filepath.IsAbs(c.EnvFrom) {
+		envFileAbsPath = path.Join(envFileAbsPath, path.Base(c.EnvFrom))
```

Comment:
I don't understand this logic at all. Why would the path of the config matter?

Reply:
This was handling the case when env_from was absolute. Your suggestion below works too, it's just handling if abs() vs if !abs()

Reply:
If `env_from` is absolute you don't need to change it at all.

Reply:
yeah I agree, I overcomplicated the logic there.